swaybar/config.c

```diff
@@ -46,6 +46,7 @@ char *parse_font(const char *font) {
 struct config *init_config() {
 	struct config *config = calloc(1, sizeof(struct config));
 	config->status_command = NULL;
+	config->plaintext_markup = true;
```

Comment:
I might be missing how this works compared to i3, but as I understand this is not possible in i3, so maybe it should default to `false` so it behaves as expected by default.